docs/Qualitaetssicherung.tex

```diff
@@ -41,6 +41,24 @@
 \section{Einleitung}
 
 \section{Änderungen in der Implementierung}
+Während der Testphase wurde einige Fehler behoben und Verhaltensweisen verändert, um OSIP benutzerfreundlicher zu machen.
+Diese Probleme wurde nicht notwendigerweise durch die Testszenarien aufgedeckt, da die Verhaltensweisen nicht so genau
```

Comment:
wurde [hier fehlt ein n] nicht [...] , da die Verhaltensweisen nicht [im Fokus des Pflichtenhefts und daher nicht ] genau spezifiziert [waren] oder die Szenarien [durch ihre Beschränkung auf Abdeckung aller Anforderungen] nicht aufdecken konnten.
Ich hätte die Erläuterung noch etwas präziser gemacht, da es für mich so klingt wie "Wir haben uns keine Mühe beim Pflichtenheft gegeben und mussten jetzt nochmal alles anpassen." (vielleicht etwas übertrieben, aber in diese Richtung geht es).